### PR TITLE
Build out basic form

### LINKED ISSUE #37: [task] Set up general testing framework
Working on this alongside #26.

### PR DESCRIPTION
**Pull request by fureigh**
This PR addresses #26 and closes #37 in the process. It does the following:
- Creates and displays a deliberately non-saving FAFSA form with `first name`, `last name` and `date of birth` fields.
- Adds a header with U.S. Government website disclaimers. (Asset TK.)
- Adds intro text.
- Fixes loading of U.S. Draft Web Design Standards.
- Sets up initial functional and unit tests.
- Updates the README accordingly.

Note that you'll need to apply the migration(s) by running `python manage.py migrate`.

### CODE REVIEW

**Comment by fureigh** [comment]
Let's handle the addition of more fields in different PRs: splitting them up by section, say.

**Comment by fureigh** [comment]
@donjo Ready for review.

Known quirks:
- U.S. flag asset TK.
- Form field labels don't yet match [@kategarklavs' specifications](https://docs.google.com/document/d/1RcVIEAepWr6vSdxahgFp0rrUVObvmYgHjZ9jlo6WpYg/edit#).

File: ffd_info_exchange/fafsa/templates/fafsa_form.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+{% load static from staticfiles %}
+
+{% block content %}
+  {% load floppyforms %}
+    <header class="usa-site-header disclaimer" role="banner">
```
Comment (fureigh) [inline ffd_info_exchange/fafsa/templates/fafsa_form.html:6]: @donjo If you wanna refactor...

Comment (donjo) [reply ffd_info_exchange/fafsa/templates/fafsa_form.html:6]: Yeah I'll break this out in a new PR. 👍

**Review by donjo — APPROVED** [review]

**Pull request merged**